### DISCUSSION TITLE
Support for stretchable images?

### DISCUSSION THREAD

**Discussion by thomasgrivet** [rnmapbox/maps#1959] [Q&A]
Hello! First of all, I would like to thank every person who contributed to this project, it has been really helpful.
I'm using the latest released version (8.6.0-beta.0) and after many tries and errors using PointAnnotation and MarkerView, I ended up using a simple ShapeSource with a SymbolLayer to render text inside a box with a simple arrow located at the bottom of said box.

<img width="191" alt="Screenshot 2022-06-07 at 14 24 50" src="https://user-images.githubusercontent.com/18561703/172378080-a075ab55-c4ce-4fee-9af7-7ccdf21439c0.png">

To do this, I created a simple .png file and I'm using the iconTextFit style key to increase / decrease the dimensions of the background image.
However, the border-radius and the arrow width also change, which doesn't look great, especially when multiple items are close to each others with various sizes.
While browsing Mapbox's official examples, I stumbled across this one: https://docs.mapbox.com/mapbox-gl-js/example/add-image-stretchable/.
This would resolve the problem I and probably others have. Can this feature be supported?

Thanks, have a nice day ☕️

**Comment by Crysp** [comment]
I'm also really need this feature. Is any plans to implement this?
